dev/NavigationView/NavigationView_InteractionTests/NavigationViewTests.cs

```diff
@@ -4366,17 +4366,22 @@ public void VerifyIconsRespectCompactPaneLength()
                 var currentStatus = new CheckBox(FindElement.ByName("MenuItemsCorrectOffset"));
 
                 checkMenuItemsButton.Click();
+                Log.Comment("Verifying compact pane length set before loading working");
                 Wait.ForIdle();
+                compactpaneCheckbox = new ComboBox(FindElement.ByName("CompactPaneLenghtComboBox")); 
```

Comment:
>CompactPaneLenghtComboBox [](start = 71, length = 25)

nit: I realize this is a previous typo, but can be fixed?

Reply:
Good idea, fixed.